trunk/SUAVE/Components/Energy/Converters/Rotor.py

```diff
@@ -682,3 +687,45 @@ def prop_vel_to_body(self):
     
     def vec_to_prop_body(self):
         return self.prop_vel_to_body()
+
+ 
+## @ingroup Components-Wings
+class Airfoil_Container(ContainerOrdered):
```

Comment:
If the append function (see above) is not unique to airfoils then you can just import ContainerOrdered